src/schema/Schema.ts

```diff
@@ -100,7 +110,8 @@ export class Schema {
       // relationship of the parent model. In this case, we'll attach any
       // missing foreign keys to the record first.
       if (key !== null) {
-        ;(parent.$fields()[key] as Relation).attach(parentRecord, record)
+        // Using optional chaining in the event of an inverse polymorphic relation that won't have a defined relation
+        ;(parent.$fields()[key] as Relation)?.attach(parentRecord, record)
```

Comment:
For `MorphTo`, the parent and child are set to the same model. This causes issues because we will expect a key that isn't going to be present on the parent model, which is why I added optional chaining here.

We will handle relating the parent to the child in the union schema callback.